code/modules/mob/living/carbon/human/species_types/android.dm

```diff
@@ -58,7 +58,7 @@
 		limb.skin_tone ||= skintone
 		limb.limb_id = SPECIES_HUMAN
 		limb.should_draw_greyscale = TRUE
-		limb.name = "human [parse_zone(limb.body_zone)]"
+		limb.name = "human [limb.plaintext_zone]"
```

Comment:
same thing here

Reply:
this is intentional, androids are supposed to have faux human limbs.